### ISSUE TITLE
Test site with NVDA

### ISSUE THREAD

**Issue by kinow** [kinow/kinoshita.eti.br#164]
https://www.nvaccess.org/about-nvda/

From https://news.ycombinator.com/item?id=26532804

**Comment by kinow** [comment]
NVDA doesn't work on Linux. Learned how to use the [Orca](https://help.gnome.org/users/orca/stable/index.html.en) screen reader [^1].

Improved the sidebar and parts of the landing page [^2] but more to come soon.

[^1]: https://techblog.wikimedia.org/2020/07/02/an-orca-screen-reader-tutorial/
[^2]: https://github.com/kinow/kinoshita.eti.br/commit/dfd0be146001509b7d3530a462998b06825d3214
